### PR TITLE
approximate design effect to adjust survey effective sample size

### PR DESCRIPTION
**Pull request by jeffeaton**
This PR adds three numeric inputs under the 'Advanced Options' tab.  This needs review in `hintr`.  Ideally new strings also need translated, but not the end of the world if not.

Thanks,
Jeff

### CODE REVIEW

File: inst/metadata/advanced_run_options.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
-{
-	"label": "t_(OPTIONS_CALIBRATION_LABEL)",
+t": "{
+	"label_(OPTIONS_CALIBRATION_LABEL)",
```
Comment (r-ash) [inline inst/metadata/advanced_run_options.json]: Oops - an accidental highlight and drag? This is the wrong syntax.

File: inst/metadata/advanced_run_options.json [hunk 2]
```diff
@@ -214,6 +214,36 @@
 					"helpText": "t_(OPTIONS_ADVANCED_LOGIT_HELP)"
 				}
 			]
+		},
+	        {
+			"label": "Approximate district survey design effect",
```
Comment (r-ash) [inline inst/metadata/advanced_run_options.json]: Can you move all the strings into the resource bundle https://github.com/mrc-ide/naomi/blob/master/inst/traduire/en-translation.json#L78

This is for each string which gets returned to the user (and so requires translation) pick a key name and use that here e.g.
```
"label": "t_(OPTIONS_DESIGN_EFFECT_LABEL)"
```

Then there should be a line in the resource bundle
```
"OPTIONS_DESIGN_EFFECT_LABEL": "Approximate district survey design effect"
```

That should be for all user facing strings here

File: inst/metadata/advanced_run_options.json [hunk 3]
```diff
@@ -214,6 +214,36 @@
...
+                       "helpText": "Approximate design effect to scale effective sample size for survey observations. Typically this will be smaller than the full survey design effect.",
+		        "controls": [
+				{
+					"name": "deff_prev",
+					"label": "Prevalence",
+					"type": "number",
+					"required": true,
+				        "value" = 1.0
```
Comment (r-ash) [inline inst/metadata/advanced_run_options.json]: Should be 
```
"value": 1.0
```

Comment (jeffeaton) [reply inst/metadata/advanced_run_options.json]: Sigh, thanks. This makes a strong case for just going to bed.

Fixed in 75406ff

File: inst/metadata/advanced_run_options.json (hunk 2 above)
Comment (jeffeaton) [reply inst/metadata/advanced_run_options.json]: Thanks. Done in 2efcab9.

At some point I thought Rich mentioned something like that everything that goes into the translation resource needs to be tested. Do I need to do anything further on this?

Comment (r-ash) [reply inst/metadata/advanced_run_options.json]: I don't know about that - everything there needs to be translated but I don't believe we're testing every string at the moment. Maybe @richfitz can comment? 

This looks fine to me - I will ask Anne to give us a translation of these strings and add that to this PR before merging.

File: inst/metadata/advanced_run_options.json [hunk 4]
```diff
@@ -214,6 +214,36 @@
 					"helpText": "t_(OPTIONS_ADVANCED_LOGIT_HELP)"
 				}
 			]
+		},
+	        {
+		       "label": "t_(OPTIONS_ADVANCED_APPROX_DEFF_LABEL)",
+                       "helpText": "t_(OPTIONS_ADVANCED_APPROX_DEFF_LABEL)",
```
Comment (r-ash) [inline inst/metadata/advanced_run_options.json]: Sorry another thing came up after putting this through our schema. You can't have help text here. Help text goes on the individual controls only.

Comment (jeffeaton) [reply inst/metadata/advanced_run_options.json]: Thanks -- removed in 7144474. (Also that is a pretty nice commit hash.)

**Pull request merged**